src/Java.Interop.Tools.Cecil/Java.Interop.Tools.Cecil/TypeDefinitionRocks.cs

```diff
@@ -92,6 +92,25 @@ public static bool IsSubclassOf (this TypeDefinition type, string typeName, IMet
 			return false;
 		}
 
+		public static bool HasJavaPeer (this TypeDefinition type, IMetadataResolver resolver)
```

Comment:
Is the idea we'd call this same method from xamarin-android as well?

So we don't have duplicate of this logic running around:

https://github.com/xamarin/xamarin-android/blob/2cbb5c98ebed30283d89e8434210415d4d5e9ded/src/Xamarin.Android.Build.Tasks/Utilities/XAJavaTypeScanner.cs#L80